qhub/render/__init__.py

```diff
@@ -86,7 +86,14 @@ def render_template(output_directory, config_filename, force=False, dry_run=Fals
             ".terraform.lock.hcl",
             "terraform.tfstate.backup",
         ],
-        ignore_directories=[".terraform"],
+        ignore_directories=[
+            ".terraform",
+            "__pycache__",
+        ],
+        deleted_paths=[
```

Comment:
i think i'm worried that defining these paths deep in the code might wake it hard to manage these deprecations. it seems that this is code style as it is. we might want to tear this out into a `deprecate.py` file or something.

Reply:
Thanks for the review @tonyfast. What do you think about the recent additions? I've added an explicit DEPREICATED_FILE_PATHS constant within `qhub/depricate.py`. Eventually I'd like to add this logic to our extension mechanism to allow for stages to depricate files.

Reply:
I'd like to keep the `deleted_paths` bit just to communicate that some files will actually be deleted.